src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ActivatorUtilities.cs

```diff
@@ -127,6 +129,13 @@ public static class ActivatorUtilities
             [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] Type instanceType,
             Type[] argumentTypes)
         {
+#if NETSTANDARD2_1_OR_GREATER || NET6_0_OR_GREATER
```

Comment:
NET6_0_OR_GREATER rather than a lower version just because that's the lowest version we still build for?  IsDynamicCodeSupported was introduced in .NET Core 3.0, I believe.

Reply:
MS.Extensions uses `NETCOREAPP` when we mean "all .NET Core versions". We don't target anything lower than net6.0.

We should be consistent here.